nerfstudio/field_components/temporal_distortions.py

```diff
@@ -30,7 +30,9 @@ class TemporalDistortion(nn.Module):
     """Apply spatial distortions as a function of time"""
 
     @abc.abstractmethod
-    def forward(self, positions: Float[Tensor, "*bs 3"], times: Optional[Float[Tensor, "1"]]) -> Float[Tensor, "*bs 3"]:
+    def forward(
+        self, positions: Float[Tensor, "*bs 3"], times: Optional[Float[Tensor, "*bs 1"]] = None
+    ) -> Optional[Float[Tensor, "*bs 3"]]:
```

Comment:
I am not a fan of returning Optional here, but I guess it is ok. 
An alternative: if times is None, don't call TemporalDistortion?

Reply:
Did the alternative 🙂